src/main/kotlin/com/example/pass_manager/repositories/ClientRepository.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.pass_manager.repositories
+
+import com.example.pass_manager.domain.MongoClient
+import org.bson.types.ObjectId
+import org.springframework.data.mongodb.repository.MongoRepository
+
+interface ClientRepository : MongoRepository<MongoClient, ObjectId> {
+    fun existsByEmailOrPhoneNumber(email: String, phoneNumber: String): Boolean
+    fun findByEmailAndPhoneNumber(email: String, phoneNumber: String): MongoClient?
```

Comment:
Let's rename `MongoClient`. 

Why? Spring boot provides `MongoClient` bean(an object to communicate with mongo), that clashes with your domain object name

Reply:
Just search for `MongoClient`(cmnd+o) class to observe the issue